### ISSUE TITLE
Not an issue, maybe an oversight? Waterlogged crops

### ISSUE THREAD

**Issue by Mowmaster** [AgriCraft/AgriCraft#1384]
agricraft-3.0.4
Waterlogged crops is a cool idea, but being able to waterlog, say carrots, doesnt seem quite right, unless maybe theres a hydroponics system to the mod that wasnt apparent from the current items available in the creative tab???

Cool idea, for things like rice or maybe cranberries and the sort, but for ground root crops, it seems a bit odd.

Also you cannot make traditional water stream to break crops farms, would be really nice if this still worked.

**Comment by InfinityRaider** [comment]
Yes you can waterlog crops, it is a growth requirement for some (kelp for instance), others won't grow underwater though.

I'll have a look at reimplementing flowing liquid uorooting crops as that is an important vanilla mechanic that I overlooked.
